model/map-jpa/src/main/resources/META-INF/users/jpa-users-changelog-1.xml

```diff
@@ -32,6 +32,9 @@ limitations under the License.
                 <constraints nullable="false"/>
             </column>
             <column name="metadata" type="json"/>
+            <column name="usernamelowercase" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Could changes in this file be placed in a separate changeset?

Reply:
Definitely, what would be a benefit to have it separated?

Reply:
Those already trying the JPA storage would not be confused by liquibase conflicts.

Reply:
True, on the other hand there is opened https://github.com/keycloak/keycloak/pull/13392 which renames column names across whole jpa map storage and it will affect all checksums as well. So we may not be able to avoid the confusion. wdyt?